src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodPublicOnly.java

```diff
@@ -130,9 +130,6 @@ public void run() {
                 privateMethod(null, null);
             }
 
-            /**
-             * Javadoc
-             */
             private String privateMethod(String a, String b) {
```

Comment:
Please do not update other Inputs with removal of code, they are weird by design.
You can put/remove violation comment only in them if required.

Reply:
@romani I want to return three violations for the anonymous method. I am able to return one but unable to return 2. Can you help me 
   ```
public Thread anonymousClassInMethod() {
        return new Thread() {
            @Override
            public void run() {
                privateMethod(null,null);
            }

            /**
            * Javadoc
            */
            private String privateMethod(String a, String b) {   //  violation '@return tag should be present and have description.'
                return null;
            }
        };
    }
```

Reply:
> Please do not update other Inputs with removal of code, they are weird by design. You can put/remove violation comment only in them if required.

Done @romani